package.json

```diff
@@ -111,7 +111,7 @@
     "lerna": "^3.16.4",
     "lodash": "^4.17.14",
     "precise-commits": "^1.0.2",
-    "prettier": "1.19.1",
+    "prettier": "1.18.2",
```

Comment:
What's the reason that we're pinning this version?

Reply:
There seems to be a bug in version 1.19.* that is blocking commits of `mdx` file changes.

Reply:
Prettier issue: https://github.com/prettier/prettier/issues/6943

Reply:
Perfect, thanks!